WordPress/Classes/ViewRelated/Notifications/Views/NoteBlockUserTableViewCell.swift

```diff
@@ -73,10 +73,11 @@ class NoteBlockUserTableViewCell: NoteBlockTableViewCell {
         nameLabel.textColor = WPStyleGuide.Notifications.blockTextColor
 
         blogLabel.font = WPStyleGuide.Notifications.blockRegularFont
-        blogLabel.textColor = WPStyleGuide.Notifications.blockSubtitleColor
+        blogLabel.textColor = WPStyleGuide.allTAllShadeGrey()
```

Comment:
This color is deprecated. We should use something else?

Reply:
Ahhhh i had no idea it was deprecated. Did that change because... having both the `Follow` button, and `Blog Title` in the same color looked way overloaded. Let's hold for design review!